framework/src/variables/MooseVariableDataBase.C

```diff
@@ -575,6 +575,7 @@ MooseVariableDataBase<OutputType>::fetchDoFValues()
         // tag is defined on problem but may not be used by a system
         // the grain tracker requires being able to read from solution vectors that we are also in
         // the process of writing :-/
+        // Note: the extra vector tags are also still not closed when a TagVectorAux uses them
```

Comment:
this is debatedly a bug for that auxkernel?
see https://civet.inl.gov/job/3300985/ to see an assert fail on "is vector closed"